src/Chartjs/Options/Tooltips.elm

```diff
@@ -42,16 +57,31 @@ type alias Tooltips =
     }
 
 
+{-| Currently unsupported!
```

Comment:
What's the point of having unsupported functions here?
Is support planned for the shortterm?